### ISSUE TITLE
Confusing Destructuring in Entity Adapter Example

### ISSUE THREAD

**Issue by SeymoreThrottle** [ngrx/platform#1160]
This example would probably be a lot more clear if the following code was refactored to not use destructuring. I wasn't used to seeing this syntax, and this particular usage threw me for a loop for a bit. It would be handy to include the snippet as an optimization somewhere in the doc, perhaps after all of the example code.

```
export const {
  // select the array of user ids
  selectIds: selectUserIds,

  // select the dictionary of user entities
  selectEntities: selectUserEntities,

  // select the array of users
  selectAll: selectAllUsers,

  // select the total user count
  selectTotal: selectUserTotal,
} = adapter.getSelectors(); 
```

**Comment by tdeschryver** [comment]
Would you like to make a pull request with your suggestion in order to make it clear?

**Comment by SeymoreThrottle** [comment]
#1161 submitted

**Comment by brandonroberts** [comment]
Closed via #1161
